### PR TITLE
Renames OKC -> OKTC

### PR DESCRIPTION
**Pull request by Kodylow**
Renames OKC -> OKTC across documentation.

### CODE REVIEW

File: api/oktc-swap-api/queries.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+The subgraph can be queried to retrieve important information about OKCSwap, pairs, tokens, transactions, users, and more. This page will provide examples for common queries.
```
Comment (zccst) [inline api/oktc-swap-api/queries.md:2]: rename: OKCSwap->Swap

**Review by zccst — COMMENTED** [review]
27 places need to be modified

File: nodes/operation/how-to-start-mainnet-node.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ After executing the exit command, if the following records appear in the log, it
     rm -rf ${EXCHAIND_PATH}/data
     cd ${EXCHAIND_PATH}
     # This image uses rocksdb data, so download the rocksdb snapshot corresponding to the mainnet.
-    wget https://okg-pub-hk.oss-cn-hongkong.aliyuncs.com/cdn/okc/snapshot/mainnet-$version-$date-$height-rocksdb.tar.gz
+    wget https://okg-pub-hk.oss-cn-hongkong.aliyuncs.com/cdn/oktc/snapshot/mainnet-$version-$date-$height-rocksdb.tar.gz
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply nodes/operation/how-to-start-mainnet-node.md:77]: done

File: nodes/operation/how-to-join-the-public-testnet.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ Note 4: If you encounter problems in the process of compiling rocksdb, please re
     exchaind init your_custom_moniker --chain-id exchain-65 --home ${EXCHAIND_PATH}# 2. download snapshot
     rm -rf ${EXCHAIND_PATH}/data
     cd ${EXCHAIND_PATH}
-    wget https://okg-pub-hk.oss-cn-hongkong.aliyuncs.com/cdn/okc/snapshot/testnet-$version-$date-$height-rocksdb.tar.gz
+    wget https://okg-pub-hk.oss-cn-hongkong.aliyuncs.com/cdn/oktc/snapshot/testnet-$version-$date-$height-rocksdb.tar.gz
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply nodes/operation/how-to-join-the-public-testnet.md:34]: done

File: nodes/operation/how-to-join-the-public-testnet.md [hunk 4]
```diff
@@ -10,15 +10,15 @@ There are two ways to start exchaind testnet node.Please note that testnet node
...
+Note 3: If this is the first time to run exchaind, we should install rocksdb first. OKTC team has provided a command to `make rocksdb` to help. You can try it in the exchain folder.
 Note 4: If you encounter problems in the process of compiling rocksdb, please refer to [this link](https://forum.okt.club/d/267-rocksdb "this link")
 
 
     # 2. Initialize exchain node configurationsexport EXCHAIND_PATH=~/.exchaind (or other directory)
     exchaind init your_custom_moniker --chain-id exchain-65 --home ${EXCHAIND_PATH}# 3. download snapshot
     rm -rf ${EXCHAIND_PATH}/data
     cd ${EXCHAIND_PATH}
-    wget https://okg-pub-hk.oss-cn-hongkong.aliyuncs.com/cdn/okc/snapshot/testnet-$version-$date-$height-rocksdb.tar.gz
+    wget https://okg-pub-hk.oss-cn-hongkong.aliyuncs.com/cdn/oktc/snapshot/testnet-$version-$date-$height-rocksdb.tar.gz
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply nodes/operation/how-to-join-the-public-testnet.md:21]: done

File: core-concepts/validator/validators-faq.md [hunk 5]
```diff
@@ -44,18 +44,18 @@ Out of all validator candidates that signaled themselves, the 21 with the highes
...
-  - It is associated with a public key `okchainvalconspub` (Get this value with `exchaind tendermint show-validator`)
+  - It is associated with a public key `oktchainvalconspub` (Get this value with `exchaind tendermint show-validator`)
   - It is generated when the node is created with exchaind init.
-- **Application key**: This key is created from `exchaincli` and used to sign transactions. Application keys are associated with a public key prefixed by `okchainpub` and an address prefixed by `okc`. Both are derived from account keys generated by `exchaincli keys add`.
+- **Application key**: This key is created from `exchaincli` and used to sign transactions. Application keys are associated with a public key prefixed by `oktchainpub` and an address prefixed by `oktc`. Both are derived from account keys generated by `exchaincli keys add`.
 
-Note: A validator’s operator key is directly tied to an application key, but uses reserved prefixes solely for this purpose: `okchainvaloper` and `okchainvaloperpub`.
+Note: A validator’s operator key is directly tied to an application key, but uses reserved prefixes solely for this purpose: `oktchainvaloper` and `oktchainvaloperpub`.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply core-concepts/validator/validators-faq.md:58]: done

File: core-concepts/validator/validators-faq.md [hunk 6]
```diff
@@ -44,18 +44,18 @@ Out of all validator candidates that signaled themselves, the 21 with the highes
...
 ### What are the different types of keys?
 
 In short, there are two types of keys:
 
 - **Tendermint Key**: This is a unique key used to sign consensus votes.
-  - It is associated with a public key `okchainvalconspub` (Get this value with `exchaind tendermint show-validator`)
+  - It is associated with a public key `oktchainvalconspub` (Get this value with `exchaind tendermint show-validator`)
   - It is generated when the node is created with exchaind init.
-- **Application key**: This key is created from `exchaincli` and used to sign transactions. Application keys are associated with a public key prefixed by `okchainpub` and an address prefixed by `okc`. Both are derived from account keys generated by `exchaincli keys add`.
+- **Application key**: This key is created from `exchaincli` and used to sign transactions. Application keys are associated with a public key prefixed by `oktchainpub` and an address prefixed by `oktc`. Both are derived from account keys generated by `exchaincli keys add`.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply core-concepts/validator/validators-faq.md:56]: done

File: core-concepts/validator/validators-faq.md [hunk 7]
```diff
@@ -44,18 +44,18 @@ Out of all validator candidates that signaled themselves, the 21 with the highes
...
-We view testnet participation as a great way to signal to the community that you are ready and able to operate a validator. You can find all relevant information about the testnet [here](/dev/quick-start/join-okc-testnet.html) and [here](https://github.com/okx/testnets).
+We view testnet participation as a great way to signal to the community that you are ready and able to operate a validator. You can find all relevant information about the testnet [here](/dev/quick-start/join-oktc-testnet.html) and [here](https://github.com/okx/testnets).
 
 ### What are the different types of keys?
 
 In short, there are two types of keys:
 
 - **Tendermint Key**: This is a unique key used to sign consensus votes.
-  - It is associated with a public key `okchainvalconspub` (Get this value with `exchaind tendermint show-validator`)
+  - It is associated with a public key `oktchainvalconspub` (Get this value with `exchaind tendermint show-validator`)
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply core-concepts/validator/validators-faq.md:54]: done

File: core-concepts/delegator/delegators-guide-cli.md [hunk 8]
```diff
@@ -217,13 +217,13 @@ exchaincli query account <yourAddress>
 // query the list of validators
 exchaincli query staking validators
 
-// query the information of a validator given their address (e.g. okchainvaloper1alq9na49n9yycysh889rl90g9nhe58lcs50wu5)
+// query the information of a validator given their address (e.g. oktchainvaloper1alq9na49n9yycysh889rl90g9nhe58lcs50wu5)
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply core-concepts/delegator/delegators-guide-cli.md:220]: done

File: api/oktc-swap-api/entities.md [hunk 9]
```diff
@@ -117,14 +117,14 @@ Burn entities are created for every emitted Burn event on the Uniswap core contr
...
-The Bundle is used as a global store of USDT. In OKCswap, 1USDT is always equal to $1,thus we use USDT as our common base token to derive  USD values of other Tokens.  
+The Bundle is used as a global store of USDT. In OKTCswap, 1USDT is always equal to $1,thus we use USDT as our common base token to derive  USD values of other Tokens.  
 | Field Name  | Value Type  |  Description  |
 |:---|:---|:---|
 |id|ID|constant 1|
 |usdPrice|BigDecimal|always equal to 1|
 # Historical Entities
 The subgraph tracks information grouped in daily buckets, using timestamps provided by contract events. These entities can be used to query things like total volume on a given day, price of a token on a given day, etc.
-## OKCswapDayData
+## OKTCswapDayData
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/entities.md]: done

File: api/oktc-swap-api/entities.md [hunk 10]
```diff
@@ -134,7 +134,7 @@ Tracks data across all pairs aggregated into a daily bucket.
 |totalVolumeUSD|BigDecimal|all time volume across all pairs in USD up to and including this day|
 |totalLiquidityUSD|BigDecimal|total liquidity across all pairs in USD up to and including this day|
 |maxStored|Int|reference used to store most liquid tokens, used for historical liquidity charts|
-|mostLiquidTokens|[TokenDayData!]|tokens with most liquidity in OKCswap|
+|mostLiquidTokens|[TokenDayData!]|tokens with most liquidity in OKTCswap|
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/entities.md]: done

File: api/oktc-swap-api/queries.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
 # Queries
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/queries.md:1]: done

File: api/oktc-swap-api/queries.md [hunk 12]
```diff
@@ -54,8 +54,8 @@ Fetch a snapshot of the current state of the pair with common values. This examp
  }
 }
 ```
-### All pairs in OKCswap
-The Graph limits entity return amounts to 1000 per query as of now. To get all pairs on OKC swapuse a loop and graphql skip query to fetch multiple chunks of 1000 pairs. The query would look like this (where skip is some incrementing variable passed into your query).
+### All pairs in OKTCswap
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/queries.md]: done

File: api/oktc-swap-api/queries.md [hunk 13]
```diff
@@ -66,7 +66,7 @@ The Graph limits entity return amounts to 1000 per query as of now. To get all p
 }
 ```
 ### Most liquid pairs
-Order by liquidity to get the most liquid pairs in OKCswap.
+Order by liquidity to get the most liquid pairs in OKTCswap.
 ```
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/queries.md:70]: done

File: api/oktc-swap-api/queries.md [hunk 14]
```diff
@@ -117,9 +117,9 @@ Day data is useful for building charts and historical views around entities. To
 }
 ```
 ## Token Data
-Token data can be fetched using the token contract address as an ID. Token data is aggregated across all pairs the token is included in. Any token that is included in some pair in OKCswap can be queried.
+Token data can be fetched using the token contract address as an ID. Token data is aggregated across all pairs the token is included in. Any token that is included in some pair in OKTCswap can be queried.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/queries.md]: done

File: api/oktc-swap-api/queries.md [hunk 15]
```diff
@@ -117,9 +117,9 @@ Day data is useful for building charts and historical views around entities. To
...
 ### Token Overview
-Get a snapshot of the current stats on a token in OKCswap. This query fetches current stats on WOKT. The allPairs field gets the first 200 pairs WOKT is included in sorted by liquidity in derived USD.
+Get a snapshot of the current stats on a token in OKTCswap. This query fetches current stats on WOKT. The allPairs field gets the first 200 pairs WOKT is included in sorted by liquidity in derived USD.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/queries.md]: done

File: api/oktc-swap-contract/factory.md [hunk 16]
```diff
@@ -1,13 +1,13 @@
 # Factory
 ## Contract info
 **Contract name**: OKCSwapFactory
-View on github:[OKCSwapFactory.sol on GitHub.](https://github.com/okx/OKCSwap/blob/main/contracts/pair/OKCSwapFactory.sol)
+View on github:[OKCSwapFactory.sol on GitHub.](https://github.com/okx/OKTCSwap/blob/main/contracts/pair/OKCSwapFactory.sol)
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-contract/factory.md]: done

File: api/oktc-swap-contract/router.md [hunk 17]
```diff
@@ -1,16 +1,16 @@
 # Router
 ## Contract info
-**Contract name**: OKCSwapRouter
-View on github:[OKCSwapRouter.sol on GitHub](https://github.com/okex/OKCSwap/blob/main/contracts/router/OKCSwapRouter02.sol).  
+**Contract name**: OKTCSwapRouter
+View on github:[OKTCSwapRouter.sol on GitHub](https://github.com/okex/OKTCSwap/blob/main/contracts/router/OKTCSwapRouter02.sol).
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-contract/router.md]: done

File: api/oktc-swap-contract/router.md [hunk 18]
```diff
@@ -1,16 +1,16 @@
...
+View on Oklink: [Router  contract on OkLink](https://www.oklink.com/en/oktc/address/0xc97b81B8a38b9146010Df85f1Ac714aFE1554343).
 ## Read functions
 ### WOKT
 `function WOKT() external pure returns (address);`
-Returns the canonical address for OKX: [WOKT Token](https://www.oklink.com/en/okc/token/0x8f8526dbfd6e38e3d8307702ca8469bae6c56c15).
+Returns the canonical address for OKX: [WOKT Token](https://www.oklink.com/en/oktc/token/0x8f8526dbfd6e38e3d8307702ca8469bae6c56c15).
 ### factory
 `function factory() external pure returns (address);`
-Returns the canonical address for [OKCSwapFactory](https://www.oklink.com/en/okc/address/0x7b9F0a56cA7D20A44f603C03C6f45Db95b31e539).
+Returns the canonical address for [OKTCSwapFactory](https://www.oklink.com/en/oktc/address/0x7b9F0a56cA7D20A44f603C03C6f45Db95b31e539).
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-contract/router.md]: done

File: building-modules/wasm-module/vmbridge.md [hunk 19]
```diff
@@ -23,9 +23,9 @@ To make your contract initiate CW20 exchange for ERC20, you must develop an EVM
 ### EVM contract rules
 Rid of the following actions in order to exchange ERC20 for CW20 tokens. (equivalent to calling the mintCW20 method of the CW20 contract, wasmAddr is the wasm contract address, recipient is the cw20 token receiving address, and amount is the number of cw20 tokens)
 ```solidity
-event __OKCSendToWasm(string wasmAddr, string recipient, uint256 amount);
+event __OKTCSendToWasm(string wasmAddr, string recipient, uint256 amount);
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply building-modules/wasm-module/vmbridge.md:26]: done

File: core-concepts/delegator/delegators-guide-cli.md [hunk 20]
```diff
@@ -217,13 +217,13 @@ exchaincli query account <yourAddress>
...
 exchaincli query staking validator <validatorAddress>
 
 // query all information of delegations and all shares recently added by a delegator (e.g. ex19n6w5l0htdgn2zwet9rtgvrzuf4a3qp4znwfcn)
 exchaincli query staking delegator <delegatorAddress>
 
-// query the information of all shares recently added to a validator (e.g. okchainvaloper1alq9na49n9yycysh889rl90g9nhe58lcs50wu5) 
+// query the information of all shares recently added to a validator (e.g. oktchainvaloper1alq9na49n9yycysh889rl90g9nhe58lcs50wu5)
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply core-concepts/delegator/delegators-guide-cli.md:226]: done

File: api/oktc-swap-api/entities.md [hunk 21]
```diff
@@ -117,14 +117,14 @@ Burn entities are created for every emitted Burn event on the Uniswap core contr
 |logIndex|BigInt|event index within transaction|
 |amountUSD|BigDecimal|derived amount of tokens sold in USD|
 ## Bundle
-The Bundle is used as a global store of USDT. In OKCswap, 1USDT is always equal to $1,thus we use USDT as our common base token to derive  USD values of other Tokens.  
+The Bundle is used as a global store of USDT. In OKTCswap, 1USDT is always equal to $1,thus we use USDT as our common base token to derive  USD values of other Tokens.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/entities.md]: done

File: api/oktc-swap-api/entities.md [hunk 22]
```diff
@@ -50,24 +50,24 @@ Information about a pair. Includes references to each token within the pair, vol
...
-|id|ID|OKX chain transaction hash|
+|id|ID|OKT Chain transaction hash|
 |blockNumber|BigInt|block transaction was mined in|
 |timestamp|BigInt|timestamp for transaction|
 |mints|[Mint]|array of Mint events within the transaction, 0 or greater|
 |burns|[Burn]|array of Burn events within transaction, 0 or greater|
 |swaps|[Swap]|array of Swap events within transaction, 0 or greater|
 ## Mint
-Mint entities are created for every emitted Mint event on the OKCswap core contracts. The Mint entity stores key data about the event like token amounts, who sent the transaction, who received the liquidity, and more. This entity can be used to track liquidity provisions on pairs.  
+Mint entities are created for every emitted Mint event on the OKTCswap core contracts. The Mint entity stores key data about the event like token amounts, who sent the transaction, who received the liquidity, and more. This entity can be used to track liquidity provisions on pairs.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/entities.md]: done

File: api/oktc-swap-api/entities.md [hunk 23]
```diff
@@ -50,24 +50,24 @@ Information about a pair. Includes references to each token within the pair, vol
...
+A user entity is created for any address that provides liquidity to a pool on OKTCswap. This entity can be used to track open positions for users. LiquidyPosition entities can be referenced to get specific data about each position.  
 | Field Name  | Value Type  |  Description  |
 |:---|:---|:---|
 |id|ID|user address|
 |liquidityPositions|[LiquidityPosition]|array of all liquidity positions user has open|
 |usdSwapped|BigDecimal|total USD value swapped|
 ## Transaction  
-Transaction entities are created for each OKX chain transaction that contains an interaction within OKCswap contracts. This subgraph tracks Mint, Burn, and Swap events on the OKCswap core contracts. Each transaction contains 3 arrays, and at least one of these arrays has a length of 1.  
+Transaction entities are created for each OKT Chain transaction that contains an interaction within OKTCswap contracts. This subgraph tracks Mint, Burn, and Swap events on the OKTCswap core contracts. Each transaction contains 3 arrays, and at least one of these arrays has a length of 1.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/entities.md]: done

File: api/oktc-swap-api/entities.md [hunk 24]
```diff
@@ -50,24 +50,24 @@ Information about a pair. Includes references to each token within the pair, vol
 |liquidityPositions|[LiquidityPosition]|rray of liquidity providers, used as a reference to LP entities|
 
 ## User  
-A user entity is created for any address that provides liquidity to a pool on OKCswap. This entity can be used to track open positions for users. LiquidyPosition entities can be referenced to get specific data about each position.  
+A user entity is created for any address that provides liquidity to a pool on OKTCswap. This entity can be used to track open positions for users. LiquidyPosition entities can be referenced to get specific data about each position.
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/entities.md]: done

File: api/oktc-swap-api/entities.md [hunk 25]
```diff
@@ -31,7 +31,7 @@ Information about a pair. Includes references to each token within the pair, vol
 | Field Name  | Value Type  |  Description  |
 |:---|:---|:---|
 |id|ID|pair contract address|
-|factory|OKCswapFactory|reference to OKCswap factory entity|
+|factory|OKTCswapFactory|reference to OKTCswap factory entity|
```
[the remark this answers is not in the document]
Comment (Kodylow) [reply api/oktc-swap-api/entities.md]: done

File: core-concepts/delegator/delegators-guide-cli.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (mazr711001) [reply core-concepts/delegator/delegators-guide-cli.md]: I not Very expert so i need Frends help me because my nollege English not Very good.And steap to do i Don't no because i 0 nollege computer

[the remark this answers is not in the document]
Comment (mazr711001) [reply core-concepts/delegator/delegators-guide-cli.md]: That why i not reply and not comment

**Review by zccst — APPROVED** [review]
